CA_french/astrophysics_fr.properties

```diff
@@ -31,7 +31,7 @@ datasetfieldtype.astroFacility.description=L'observatoire ou l'installation o
 datasetfieldtype.astroInstrument.description=L'instrument utilis� pour recueillir les donn�es.
 datasetfieldtype.astroObject.description=Les objets astronomiques repr�sent�s dans les donn�es (Il faut donner, de pr�f�rence, des noms reconnaissables par SIMBAD.)
 datasetfieldtype.resolution.Spatial.description=La r�solution spatiale (angulaire) typique des observations, en degr�s d�cimaux.
-datasetfieldtype.resolution.Spectral.description=La r�solution spectrale typique des observations, indiqu�e sous forme de ratio ?/??.
+datasetfieldtype.resolution.Spectral.description=La r�solution spectrale typique des observations, indiqu�e sous forme de ratio \u03bb/\u0394\u03bb.
```

Comment:
Shouldn't this file be in UTF-8? See: https://github.com/scholarsportal/dataverse/pull/30/commits/98e9ac2ab33f533f04e1f515c48fd6b00a26a6f0

Reply:
In fact English version in in UTF-8 : https://github.com/GlobalDataverseCommunityConsortium/dataverse-language-packs/blob/dataverse-v4.10.1/en_US/astrophysics.properties